save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -192,7 +192,7 @@ class DownloadFilesController(
             part.filename()
```

Comment:
I though we decide to remote filename from avatar name?

Reply:
We then decided to leave the full path to the avatar in the database and not change the name

Reply:
then you need to update this path in database when upsert happens

Reply:
Yes, it is updated